buildpacks/ruby/src/steps/detect_rake_tasks.rs

```diff
@@ -1,4 +1,9 @@
-use crate::build_output::{self, Section};
+#[allow(clippy::wildcard_imports)]
```

Comment:
(optional) I'm not really a fan of wildcard imports fwiw

Reply:
I removed all `section_log::*` imports, but left the `build_log::*` ones as those are needed for the API (the BuildLog struct implements several traits that must be imported for it to function correctly).